src/app/cli/src/init/graphql_queries.ml

```diff
@@ -127,6 +127,35 @@ query pendingSnarkWork {
   }
 |}]
 
+module Pending_snark_work_range =
+[%graphql
+{|
+query pendingSnarkWorkRange {
+  pendingSnarkWork {
+    workBundle {
+      source_first_pass_ledger_hash: sourceFirstPassLedgerHash
+      target_first_pass_ledger_hash: targetFirstPassLedgerHash
+      source_second_pass_ledger_hash: sourceSecondPassLedgerHash
+      target_second_pass_ledger_hash: targetSecondPassLedgerHash
+      fee_excess: feeExcess {
+        feeTokenLeft
+        feeExcessLeft {
+          sign
+          feeMagnitude
+        }
+        feeTokenRight
+        feeExcessRight {
+          sign
+          feeMagnitude
+        }
+      }
+      supply_increase : supplyIncrease
+      work_id: workId
+      }
+    }
+  }
+|}]
+
```

Comment:
i think should also be deleted for now, until/if the client command is added back

Reply:
Right, deleted